plugins/extractors/maxcompute/client/client.go

```diff
@@ -71,3 +83,34 @@ func (*Client) GetTableSchema(_ context.Context, table *odps.Table) (string, *ta
 	}
 	return table.Type().String(), &tableSchema, nil
 }
+
+func (c *Client) GetTablePreview(_ context.Context, partitionValue string, table *odps.Table, maxRows int) ([]string, *structpb.ListValue, error) {
+	if table.Type().String() == tableTypeView {
+		c.log.Debug("skipping preview for view", "table", table.Name())
+		return nil, nil, nil
+	}
+
+	records, err := c.tunnel.Preview(table, partitionValue, int64(maxRows))
+	if err != nil {
+		c.log.Error("failed to preview table", "table", table.Name(), "error", err)
+		return nil, nil, err
+	}
+
+	columnNames := make([]string, len(records))
+	for i, col := range records {
+		columnNames[i] = col.String()
+	}
```

Comment:
have we test this in local? it's giving wrong result, please check

Reply:
It was tested mas,
what is the expected output mas?

Reply:
it should returning the list of the column name, if you check previously it showing the value instead